modules/campaignion_donation_type/campaignion_donation_type.features.field_instance.inc

```diff
@@ -111,6 +111,60 @@ function campaignion_donation_type_field_default_field_instances() {
     ),
   );
 
+  // Exported field_instance: 'node-donation-field_form_intro'.
+  $field_instances['node-donation-field_form_intro'] = array(
+    'bundle' => 'donation',
+    'custom_add_another' => '',
+    'custom_remove' => '',
+    'default_value' => array(
+      0 => array(
+        'value' => '',
+      ),
+    ),
+    'deleted' => 0,
+    'description' => '',
```

Comment:
Should we add a description that states where/how this is displayed?

Reply:
I tried but it looks forlorn in ae-admin, not obvious which field it belongs to. I’d hope the title is clear enough?

Reply:
Ok, then good as is. ;-)